2022-01-09 17:19:15,467 INFO    MainThread:12908 [wandb_setup.py:_flush():71] setting env: {}
2022-01-09 17:19:15,467 INFO    MainThread:12908 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-09 17:19:15,468 INFO    MainThread:12908 [wandb_init.py:_log_setup():371] Logging user logs to /content/tbcnn/wandb/run-20220109_171915-bu1xk3ti/logs/debug.log
2022-01-09 17:19:15,468 INFO    MainThread:12908 [wandb_init.py:_log_setup():372] Logging internal logs to /content/tbcnn/wandb/run-20220109_171915-bu1xk3ti/logs/debug-internal.log
2022-01-09 17:19:15,468 INFO    MainThread:12908 [wandb_init.py:init():404] calling init triggers
2022-01-09 17:19:15,468 INFO    MainThread:12908 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-09 17:19:15,468 INFO    MainThread:12908 [wandb_init.py:init():460] starting backend
2022-01-09 17:19:15,468 INFO    MainThread:12908 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-09 17:19:15,491 INFO    MainThread:12908 [backend.py:ensure_launched():216] starting backend process...
2022-01-09 17:19:15,506 INFO    MainThread:12908 [backend.py:ensure_launched():222] started backend process with pid: 12921
2022-01-09 17:19:15,514 INFO    MainThread:12908 [wandb_init.py:init():469] backend started and connected
2022-01-09 17:19:15,531 INFO    MainThread:12908 [wandb_init.py:init():533] updated telemetry
2022-01-09 17:19:15,603 INFO    MainThread:12908 [wandb_init.py:init():563] communicating current version
2022-01-09 17:19:16,812 INFO    MainThread:12908 [wandb_init.py:init():568] got version response 
2022-01-09 17:19:16,812 INFO    MainThread:12908 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-09 17:19:17,048 INFO    MainThread:12908 [wandb_init.py:init():606] starting run threads in backend
2022-01-09 17:19:20,155 INFO    MainThread:12908 [wandb_run.py:_console_start():1810] atexit reg
2022-01-09 17:19:20,155 INFO    MainThread:12908 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-09 17:19:20,156 INFO    MainThread:12908 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-09 17:19:20,158 INFO    MainThread:12908 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-09 17:19:20,159 INFO    MainThread:12908 [wandb_init.py:init():633] run started, returning control to user process
2022-01-09 17:19:26,272 INFO    MainThread:12908 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-09 17:19:26,273 INFO    MainThread:12908 [wandb_run.py:_restore():1752] restore
2022-01-09 17:19:28,713 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 686
}

2022-01-09 17:19:28,967 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 686
}

2022-01-09 17:19:29,076 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 13752
}

2022-01-09 17:19:29,179 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 13752
}

2022-01-09 17:19:29,281 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 13752
}

2022-01-09 17:19:29,384 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}

2022-01-09 17:19:29,486 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}

2022-01-09 17:19:29,588 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}

2022-01-09 17:19:29,691 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}

2022-01-09 17:19:29,793 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}

2022-01-09 17:19:30,371 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}

2022-01-09 17:19:30,847 INFO    MainThread:12908 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13752
  total_bytes: 13752
}
local_info {
}

2022-01-09 17:19:32,602 INFO    MainThread:12908 [wandb_run.py:_append_files():2180] logging synced files
